packages/block-library/src/pattern/index.php

```diff
@@ -41,7 +41,16 @@ function render_block_core_pattern( $attributes ) {
 	}
 
 	$pattern = $registry->get_registered( $slug );
-	return do_blocks( $pattern['content'] );
+	$content = $pattern['content'];
+
+	if ( gutenberg_is_experiment_enabled( 'gutenberg-auto-inserting-blocks' ) ) {
```

Comment:
Does this need a check to see if the function exists before calling `gutenberg_is_experiment_enabled`? Being in `block-library`, I think this file winds up getting bundled for core releases, so wondered if it needs a defensive check?

Reply:
Ah, great point! Funny enough, I think we're actually about to introduce a linter against `gutenberg_` prefixes in the block library 😅  so I better inline this function (as is done elsewhere IIRC).

Reply:
b610c373ed